app/workers/rapid_ready_for_decision/rrd_processor.rb

```diff
@@ -93,18 +76,36 @@ def patient_info
     end
 
     def icn
-      account_record = account
-      raise AccountNotFoundError, "for user_uuid: #{form526_submission.user_uuid} or their edipi" unless account_record
+      account_records = accounts
```

Comment:
I don't understand why this variable is needed

Reply:
I'm not memoizing `accounts` and I don't want to DB query for the account more than needed within this `icn` method.